test/unit/attributes/attribute_test.rb

```diff
@@ -225,4 +225,16 @@
       attribute.send(:valid_numeric?, '1/2').must_equal false
     end
   end
+
+  describe 'to_ruby method' do
+    it 'generates the code for the attribute' do
+      attribute = Inspec::Attribute.new('application_port', description: 'The port my application uses', value: 80)
+      attribute.to_ruby.must_equal <<-RUBY.chomp
```

Comment:
This is just a preference, but I find it is easier to maintain (as well as handle failures) when the individual things you are looking for are split out.  It also makes it less susceptible to whitespace changes. Like this:

ruby_code = attribute.to_ruby
ruby_code.must_include "attribute('application_port'"   # Should have the declaration
ruby_code.must_include 'value: 80'   # We care that we use the 'value' attribute
ruby_code.must_include 'default: 80'   # We care that we use the 'default' attribute

Reply:
I think it's generally better to not get tripped up by whitespace changes - but in this case isn't correct whitespacing part of the responsibility of the method?